### ISSUE TITLE
Feature Request: Collapsible Album View

### ISSUE THREAD

**Issue by Alzarath** [owncloud/music#840]
When you have a bunch of albums, things can become awfully inconvenient to navigate. I believe a way to collapse the album art and items in album view would go far in making things easier. [An example.](https://imgur.com/a/T6bJ161)

**Comment by RandomBloke85** [comment]
Or maybe an Artist filter on the left side above Albums?

**Comment by Alzarath** [comment]
New views would be nice, but as stated in #826 they appear to be difficult to add. I was thinking the ability to collapse what's there might be an easier task, at least in the meantime.

**Comment by Alzarath** [comment]
I learned a little JS and created a [GreaseMonkey script](https://greasyfork.org/en/scripts/424726-collapsible-nextcloud-music) in the meantime, for anyone interested.

**Comment by paulijar** [comment]
After giving this some thought, and inspired by the monkey patch above (or its screen shots anyway), I believe that something like this could be implemented with a quite reasonable amount of work. I would still like to show the album covers also in the "compact" view, like this:
![image](https://user-images.githubusercontent.com/8565946/114194509-417fd300-9958-11eb-9bb5-52957352e616.png)

I was thinking to overlay a `+` or `-` icon on top of the album cover on hover, and this would act as the expand/collapse button.

Changing between the "normal" and "compact" layouts could happen via a "..." menu attached to the "Albums" navigation item in the left pane.

**Comment by paulijar** [comment]
The feature has now been merged to master with the commit 31f3a4da. Hence, it will be included in the next release (v1.2.0) once that is published, probably within a couple of weeks.

This is how the final solution looks like:
![image](https://user-images.githubusercontent.com/8565946/114278394-474ce580-9a38-11eb-8520-68f4505d56d3.png)

In addition to collapsing the tracks and shrinking the cover art, I have reduced the font size on the artists and album titles as well as margins between the albums.. Furthermore, the column width was reduced enough to fit the 4th column on 1920p display with 100% zoom.
